src/Symfony/Component/PropertyAccess/PropertyAccessorBuilder.php

```diff
@@ -122,12 +128,41 @@ public function getCacheItemPool()
     }
 
     /**
+     * Sets a naming strategy.
```

Comment:
useless IMHO :)

Reply:
to be consistent with the other methods, I added the comment. but i'm fine to remove it.